src/node/rpc/member_frontend.h

```diff
@@ -946,6 +946,24 @@ namespace ccf
         .set_openapi_hidden(true)
         .install();
 
+      using JWTKeyMap = std::map<JwtKeyId, Cert>;
+
+      auto get_jwt_keys = [this](auto& ctx, nlohmann::json&& body) {
+        auto keys = ctx.tx.template ro<JwtPublicSigningKeys>(
+          ccf::Tables::JWT_PUBLIC_SIGNING_KEYS);
+        JWTKeyMap kmap;
+        keys->foreach([&kmap](const auto& kid, const auto& kpem) {
+          kmap[kid] = kpem;
+          return false;
+        });
+
+        return make_success(kmap);
+      };
+      make_endpoint(
+        "jwt_keys/all", HTTP_GET, json_adapter(get_jwt_keys), no_auth_required)
```

Comment:
Why not just `jwt_keys`?

Reply:
No strong reason, other than there's precedent for /all elsewhere it seemed like it left more room for other GET endpoints at the same level (/issuers etc).

Happy to defer to a REST-qualified expert :)